node/src/main/scala/org/bitcoins/node/networking/peer/DataMessageHandler.scala

```diff
@@ -322,8 +327,8 @@ object DataMessageHandler {
   type OnTxReceived = Transaction => Future[Unit]
 
   /** Callback for handling a received compact block filter */
-  type OnCompactFilterReceived =
-    (DoubleSha256Digest, GolombFilter) => Future[Unit]
+  type OnCompactFiltersReceived =
+    (Vector[DoubleSha256Digest], Vector[GolombFilter]) => Future[Unit]
```

Comment:
Shouldn't this be `Vector[(DoubleSha256Digest, GolombFilter)]`? The way this is implemented currently means that the two vector's sizes could be different